cuda/private/macros/cuda_test.bzl

```diff
@@ -0,0 +1,25 @@
+load("//cuda/private:rules/cuda_library.bzl", "cuda_library")
+
+def cuda_test(
+        name,
+        srcs = [],
+        copts = [],
+        visibility = ["//visibility:private"],
```

Comment:
This is not respecting the package level default visibility, this should be unset if not specified.